src/phpDocumentor/Bootstrap.php

```diff
@@ -170,7 +170,9 @@ public function findVendorPath($baseDir = __DIR__)
 
                     return $rootCandidate . '/' . $relativeVendorDir;
                 }
-            } while (is_dir($rootCandidate));
+            } while (is_dir($rootCandidate)
+                && substr_count($rootCandidate, '/..') <= substr_count($rootCandidate, '/') / 2
```

Comment:
This check is needed because, at least during Bootstrap::testFindVendorPathComposerInstalledWithNestedVendorDirCanFail, is_dir returns true for paths that are shallower than the root; e.g. is_dir("vfs://root/../..") == true